### ISSUE TITLE
Enqueue Failure

### ISSUE THREAD

**Issue by christoff-1992** [appium/appium#7704]
## The problem

When running test's since using the XCUITest / WebDriverAgent at random intervals the following error occurs.

> Enqueue Failure: UI Testing Failure - Failure fetching attributes for element <XCAccessibilityElement: 0x7fce31ea9fc0> Device element: Error Domain=XCTestManagerErrorDomain Code=13 "Error copying attributes -25202" UserInfo={NSLocalizedDescription=Error copying attributes -25202} <unknown> 0 1

## Environment

* Appium version - 1.6.3
* Last Appium version that did not exhibit the issue - Pre 1.6.0
* MacOS Sierra - Version 10.12.2 (iMac, Late 2013)
* Node.js version v6.3.1
* Occurs on iOS 9.3 -> 10.2
* Both
* Appium CLI

## Details

The issue doesn't appear to have a particular cause and is very intermittent, However with 400+ test's, not been able to run them reliably is an issue.

When the issue occur's the console will constantly log the same line mentioned above, until it manages to sort it's self out. In most circumstances however the only way to rectify the issue is to kill the process and start Appium again via console.

The issue may be related to the WebDriverAgent, or even a bug in iOS if this is the case please let me know and I will close the issue and log retrospectively with the correct team. (WDA/Apple)

- More than happy to give any further detail's I can just let me know what you need, or if you wan't me to test out a few theories, I will keep investigating and will update as needed.

## Link to Appium logs

[https://gist.github.com/christoff-1992/612aa7f4ebc87d0df888dff90a5d32d3](https://gist.github.com/christoff-1992/612aa7f4ebc87d0df888dff90a5d32d3)

## Code To Reproduce Issue [ Good To Have ]

Private Repository - But in this circumstance there is no particular cause to the issue, it's random sometimes the test will run with no failures.

Thanks

**Comment by mykola-mokhnach** [comment]
Known [WDA issue](https://github.com/facebook/WebDriverAgent/issues/372)

**Comment by christoff-1992** [comment]
Thanks mykola-mokhnach
